### PR TITLE
Changes in the gameOfSticks.py

### PR DESCRIPTION
**Pull request by Silver-Taurus**
I have done some changes, see if you like it or not.

### CODE REVIEW

File: gameOfSticks.py [hunk 1]
```diff
@@ -1,6 +1,3 @@
-#! /usr/bin/env python3
-
-from __future__ import print_function
```
Comment (utkarsh2102) [inline gameOfSticks.py]: Why is this removed? :o

Comment (utkarsh2102) [inline gameOfSticks.py:3]: Why is this removed? :o

File: gameOfSticks.py [hunk 2]
```diff
@@ -1,6 +1,3 @@
-#! /usr/bin/env python3
```
Comment (utkarsh2102) [inline gameOfSticks.py:1]: Why is this removed? :o

**Review by utkarsh2102 — CHANGES_REQUESTED** [review]

File: gameOfSticks.py (hunk 1 above)
Comment (Silver-Taurus) [reply gameOfSticks.py:3]: For me it was of no use, so I just removed while copying the code

File: gameOfSticks.py (hunk 2 above)
Comment (utkarsh2102) [reply gameOfSticks.py:1]: All scripts under Linux execute using the interpreter specified on a first line. It is nothing but the absolute path to the interpreter.

File: gameOfSticks.py (hunk 1 above)
Comment (utkarsh2102) [reply gameOfSticks.py:3]: Run `pylint` on your local machine?
See what result it yields?
Post the score here? :P

File: gameOfSticks.py (hunk 2 above)
Comment (Silver-Taurus) [reply gameOfSticks.py:1]: ohk

File: gameOfSticks.py (hunk 1 above)
Comment (Silver-Taurus) [reply gameOfSticks.py:3]: 7.69

Comment (Silver-Taurus) [reply gameOfSticks.py:3]: it showed me where i lost points XD

**Review by utkarsh2102 — APPROVED** [review]
Thank you :)
